src/scancode/cli.py

```diff
@@ -131,6 +131,29 @@ def print_about(ctx, param, value):
 
     scancode -f json -l -c samples/zlib/ > scan.json
 
+Scan a directory while ignoring a single file. Print scan results to stdout as JSON:
```

Comment:
It will be ignored anywhere in the scanned tree. I'm sorry I don't fully understand what meant by this `is the glob applied to the path as a string? or to the filenames only?`.
Glob patterns can be applied to path and filenames i.e */src/test/* and *.doc both will work as expected (if that's what your were asking) :)